src/app/[locale]/onboarding/loading/page.tsx

```diff
@@ -66,14 +66,20 @@ export default function Page() {
   const nextID = getNextID(selectedID);
 
   useEffect(() => {
-    if (
+    const isLockedOrErrored =
+      statusQuery.data?.processingStep === "LOCKED" ||
```

Comment:
@Androz2091 does being LOCKED always mean it's errored?

Reply:
No, it means that the processing of the package has not started yet

Reply:
Tell me what you think but I think we shouldn't disable polling if it's locked because it will prevent us from knowing how the status evolves

Reply:
Yes, definitely

Reply:
Nevermind, I am just going to test it locally